homeassistant/components/light/zwave.py

```diff
@@ -88,14 +88,8 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
 
     value.set_change_verified(False)
 
-    if node.has_command_class(zwave.const.COMMAND_CLASS_SWITCH_COLOR):
-        try:
-            add_devices([ZwaveColorLight(value)])
-        except ValueError as exception:
-            _LOGGER.warning(
-                "Error initializing as color bulb: %s "
-                "Initializing as standard dimmer.", exception)
-            add_devices([ZwaveDimmer(value)])
```

Comment:
I think this will only remove the scenario that the colorbulb will not be added as a normal dimmer, but all others will be initialized under the else. I might be wrong. 
How is the RGB bulbs added today in HA? as 2 seperate entitys, 1 containing the dimmer, and one the rgb values, or just 1 entity with both?